scipeds/data/enums.py

```diff
@@ -88,7 +88,7 @@ class AwardLevel(str, Enum):
         associates (str): Associate's degree
         gt2_lt4 (str): Award of at least 2 but less than 4 academic years
         bachelors (str): Bachelor's degree
-        postbac (str): Postbaccalaureate certificate
+        postbac (str): Post-baccalaureate certificate
```

Comment:
Let's not hyphenate this one ([wikipedia](https://en.wikipedia.org/wiki/Postbaccalaureate_program) agrees)